docs/website/docs/examples/transformers/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Enriching loaded data with transformers
+description: Learn how to use dlt transformers and how to speed up your loads with parallelism
+keywords: [transformers, parallelism, example]
+---
+
+import Header from '../_examples-header.md';
+
+<Header 
+    intro="In this example you will learn how load a list of pokemone from the pokeapi and with the help of dlt transformers
+    automatically query additional data per retrieved pokemon. You will also learn how to harness parallelism with futures."
+    slug="transformer" />
+
+
+## Using transformers with the pokemon api
+
+For this example we will be loading data from the [Poke Api](https://pokeapi.co/). We will load a list of pokemon from the
```

Comment:
Look what @AstrakhantsevaAA did. 3-5 bullet points with highlights
- we create two transformers that are connected to a single list resource with |
- we load in parallel using requests
- we load in parallel using async
- we configure parallelism in config.toml
- we deselect pokemon list by default to not load this in database
provide relevant docs links